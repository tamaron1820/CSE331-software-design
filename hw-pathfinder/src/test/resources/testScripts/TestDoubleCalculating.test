# Test of Dijkstra's algorithm for calculating the decimal point in finding a path

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2 2.5567
AddEdge graph1 n2 n3 3.6765

ListChildren graph1 n1
ListChildren graph1 n2

FindPath graph1 n1 n3