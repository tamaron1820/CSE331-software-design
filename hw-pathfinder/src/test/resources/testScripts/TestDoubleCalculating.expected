# Test of Dijkstra's algorithm for calculating the decimal point in finding a path

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge 2.557 from n1 to n2 in graph1
added edge 3.677 from n2 to n3 in graph1

the children of n1 in graph1 are: n2(2.5567)
the children of n2 in graph1 are: n3(3.6765)

path from n1 to n3:
n1 to n2 with weight 2.557
n2 to n3 with weight 3.677
total cost: 6.233
